test/built-ins/String/prototype/isWellFormed/returns-boolean.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2022 Jordan Harband. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-String.prototype.at
+description: >
+  The method should return an Iterator instance.
+info: |
+  String.prototype.isWellFormed ( )
+
+  1. Let O be ? RequireObjectCoercible(this value).
+  2. Let S be ? ToString(O).
+  3. Return IsStringWellFormedUnicode(S).
+
+features: [String.prototype.isWellFormed]
+---*/
+assert.sameValue(typeof String.prototype.isWellFormed, 'function');
+
+assert.sameValue('\uD800'.isWellFormed(), false, 'a string with a lone surrogate is not well-formed');
+assert.sameValue('💩'.isWellFormed(), true, 'a surrogate pair is well-formed');
+assert.sameValue('abc'.isWellFormed(), true, 'a normal string is well-formed');
```

Comment:
```suggestion
assert.sameValue('abc'.isWellFormed(), true, 'a latin-1 string is well-formed');
```

I'd refrain from using terms like "normal" when it comes to culturally significant things such as language.